.travis.yml

```diff
@@ -35,6 +35,7 @@ before_install:
     conda config --add channels conda-forge --force
     conda config --set channel_priority strict
     conda config --set safety_checks disabled
+    conda install mamba
```

Comment:
Let's add it to the requirements-dev.txt so people don't have to dig into our CIs to run the tests.

Reply:
Mostly since mamba is not pip installable atm, so want to not force that, since it will skip that test if mamba isn't around

Reply:
I always forget about pip ;-p

But do we care about that in a tool that is all about conda?